euxfel_h5tools/test_lib.py

```diff
@@ -1,7 +1,7 @@
 import os
+import pytest
```

Comment:
`pytest ` is not part of the standard library, so it should not be here with the stdlib imports